src/includes/class-utils.php

```diff
@@ -247,36 +249,40 @@ public static function normalize_xml_character_entities( string $input ): string
 				'/&cent;/i',                  // Cent sign.
 				'/&curren;/i',                // Currency sign.
 				'/&[a-z]+;/i',                // Unknown/unhandled entities.
-				'/&#MYAMP;/i',                // Add ampersand back.
+				'/&#KEEPAMP;/i',              // Add ampersand back.
+				'/&#KEEPLT;/i',               // Add less-than back.
+				'/&#KEEPGT;/i',               // Add less-than back.
 				'/[ ]{2,}/',                  // Runs of spaces, post-handling.
 			);
 			/**
 			 * List of pattern replacements corresponding to patterns searched.
 			 */
 			$plain_replace = array(
-				'',           // Non-legal carriage return.
-				' ',          // Non-breaking space.
-				'"',          // Double quotes.
-				"'",          // Single quotes.
-				'>',          // Greater-than.
-				'<',          // Less-than.
-				'&#MYAMP;',   // Ampersand.
-				'&#169;',     // Copyright.
-				'&#8482;',    // Trademark.
-				'&#174;',     // Registered.
-				'--',         // mdash.
-				'-',          // ndash.
-				'*',          // Bullet.
-				'£',          // Pound sign.
-				'€',          // Euro sign. € ?.
-				'$',          // Dollar sign.
-				'&#x192;',    // Function sign.
-				'&#xa5;',     // Yen sign.
-				'&#xa2;',     // Cent sign.
-				'&#xa4;',     // Currency sign.
-				'',           // Unknown/unhandled entities.
-				'&amp;',      // Add ampersand back.
-				' ',          // Runs of spaces, post-handling.
+				'&#KEEPAMP;',    // Ampersand.
```

Comment:
What does the `KEEP` keyword do in this replacement? 🤔

Reply:
Nothing :) This could be any other string (or could even be removed). Notice that the previous expression `'/&[a-z]+;/i'` removes all unhandled entities and we thus need to replace the entities we wish to keep with a different string that won't be removed. This is achieved with the `#` character. The `KEEP` in the string merely functions as a slightly more expressive way to demonstrate that the value of the string is arbitrary and could be any chosen value.

Reply:
Thanks for clarifying.